### PR TITLE
Oa display documentation

### PR DESCRIPTION
**Pull request by thaisbendixen**
Pull Request
============

> Tracked in JIRA by https://up42.atlassian.net/browse/UP-5008?atlOrigin=eyJpIjoiZjMzNjFhZjNmMDY1NDgwN2I3MDgzZTc3MTk5ZjUyYTIiLCJwIjoiaiJ9

### Scope of the PR

Documentation for OneAtlas Pleiades and SPOT Display products.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

**Review by teoEnache — APPROVED** [review]
LGTM

File: source/up42-blocks/data/pleiades-display.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+Pléiades Display (Download)
+===========================
+For more information, please read the `block description <https://marketplace.up42.com/block/6ac78530-6200-42c1-b6a9-381a9f69d8ef>`_.
+
+Block type: ``DATA``
+
+This block provides pansharpened Pléiades HR imagery intersecting a given bounding box or AOI. The block outputs a DIMAP file and will store the AOI within the output feature geometry.
+
+The available output bands of the Pleiades block are: red, green, blue and near infrared. Pansharpened Pléiades HR imagery has a spatial resolution of 0.5x0.5m. The multispectral bands have a resolution of 2x2m.
```
Comment (perusio) [inline source/up42-blocks/data/pleiades-display.rst:15]: This block delivers pansharpened imagery also AFAIK. So maybe rephrase it along those lines.

File: source/up42-blocks/data/spot-display.rst [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+SPOT 6/7 Display (Download)
+===========================
+For more information, please read the `block description <https://marketplace.up42.com/block/3e24ed57-b07f-4d44-9957-5d5c0a457239>`_.
+
+Block type: ``DATA``
+
+This block provides pansharpened SPOT imagery intersecting a given bounding box or AOI. The block outputs a DIMAP file and will store the AOI within the output feature geometry.
+
+The available output bands of the SPOT block are: red, green, blue and near infrared. SPOT Pansharpened imagery has a spatial resolution of 2x2m. The multispectral bands have a resolution of 8x8m.
```
Comment (perusio) [inline source/up42-blocks/data/spot-display.rst:15]: Ditto.

**Review by perusio — CHANGES_REQUESTED** [review]
Please see below. Can we add something also about DIMAP -> GeoTIFF conversion? 🤔
Obrigado.

**Review by teoEnache — APPROVED** [review]
LGTM

File: source/up42-blocks/data/pleiades-display.rst [hunk 3]
```diff
@@ -20,6 +20,7 @@ The available output bands of the Pleiades block are: red, green, blue and near
 
 .. tip::
 
+   The DIMAP format may be converted into GeoTiff format using the `DIMAP -> GeoTIFF conversion block <https://marketplace.up42.com/block/c94bb4cd-8ee2-40df-ba76-d332b8f48c6a>`_.
    Find out more about the DIMAP image format `here <https://www.intelligence-airbusds.com/en/8722-the-dimap-format>`_. DIMAP is a GDAL supported `raster format <https://gdal.org/drivers/raster/dimap.html>`_.
```
Comment (perusio) [inline source/up42-blocks/data/pleiades-display.rst:24]: GeoTIFF.

File: source/up42-blocks/data/spot-display.rst [hunk 4]
```diff
@@ -20,7 +20,8 @@ The available output bands of the SPOT block are: red, green, blue and near infr
 
 .. tip::
 
-	 Find out more about the DIMAP image format `here <https://www.intelligence-airbusds.com/en/8722-the-dimap-format>`_. DIMAP is a GDAL supported `raster format <https://gdal.org/drivers/raster/dimap.html>`_.
+    The DIMAP format may be converted into GeoTiff format using the `DIMAP -> GeoTIFF conversion block <https://marketplace.up42.com/block/c94bb4cd-8ee2-40df-ba76-d332b8f48c6a>`_.
```
Comment (perusio) [inline source/up42-blocks/data/spot-display.rst:23]: GeoTIFF.

**Review by perusio — COMMENTED** [review]
I will change the GeoTiff to GeoTIFF here and merge it.
